rasgoql/rasgoql/primitives/rendering.py

```diff
@@ -314,6 +314,28 @@ def _set_final_select_statement(
     return sql
 
 
+def get_columns(source_table: str, running_sql: str = None, dw: 'DataWarehouse' = None) -> str:
+    if not running_sql:
+        database, schema, table_name = source_table.split('.')
+        query_string = f"""
```

Comment:
I think we may need to punt the actual SQL that gets run to the dw classes themselves. I believe we ran into a problem where not all dw types have access to an information_schema (@szaghikian-rasgo  could confirm). There may be a method on the classes already to get columns from a table, if not, we may need to add one.

Reply:
This should be fine. The aberrant case was Postgres, and the issue wasn't that you couldn't hit the information schema, but that the best way to check if a table exists in Postgres was to use `pg_catalog`. This query should still work fine for the DWs we currently support